pyproject.toml

```diff
@@ -8,7 +8,7 @@ authors = [{ name = "Andreas Mueller", email = "t3kcit+wordcloud@gmail.com" }]
 description = "A little word cloud generator"
 readme = { file = "README.md", content-type = "text/markdown" }
 requires-python = ">=3.7"
-license = { text = "MIT License" }
+license = "MIT"
```

Comment:
Looks like this new spec requires a newer version of `setuptools`, which does not support Python 3.7.

I think it's time to drop 3.7, but for the time being, I think we can put an upper bound on, `setuptools<=68.0.0`, in `[build-system]`.